packages/neos-ui/src/Sagas/UI/Hotkeys/index.js

```diff
@@ -0,0 +1,43 @@
+import Mousetrap from 'mousetrap';
+import {take, race} from 'redux-saga/effects';
+import {actionTypes} from '@neos-project/neos-ui-redux-store';
+import {getGuestFrameDocument} from '@neos-project/neos-ui-guest-frame/src/dom';
+
+export function * handleHotkeys({globalRegistry, store}) {
+    yield take(actionTypes.System.READY);
+
+    const hotkeyRegistry = globalRegistry.get('hotkeys');
+    const items = hotkeyRegistry.getAllAsList();
+    let mousetrapPaused = false;
+
+    for (let i = 0; i < items.length; i++) {
```

Comment:
Just a style nitpick, but we strongly prefer `forEach` to loops... You may ignore this comment.